read.ts

```diff
@@ -0,0 +1,71 @@
+import "@frequency-chain/api-augment";
+import { getEndpoint, getFrequencyAPI } from "./services/connect";
+
+import stringify from "json-stringify-pretty-compact";
+
+import broadcast from "./dsnp/broadcast";
+import graphChange from "./dsnp/graphChange";
+import profile from "./dsnp/profile";
+import reaction from "./dsnp/reaction";
+import reply from "./dsnp/reply";
+import tombstone from "./dsnp/tombstone";
+import update from "./dsnp/update";
+
+const nameAndSchema = [
+  ["dsnp.broadcast", JSON.stringify(broadcast)],
+  ["dsnp.profile", JSON.stringify(profile)],
+  ["dsnp.reaction", JSON.stringify(reaction)],
+  ["dsnp.reply", JSON.stringify(reply)],
+  ["dsnp.tombstone", JSON.stringify(tombstone)],
+  ["dsnp.update", JSON.stringify(update)],
+  ["dsnp.graphChange", JSON.stringify(graphChange)],
+];
+
+const read = async () => {
+  const api = await getFrequencyAPI();
+
+  const { specName, specVersion } = await api.runtimeVersion;
+  const clientVersion = await api.rpc.system.version();
+  const endpointUrl = getEndpoint();
+
+  const latestBlockNumber = await api.query.system.number();
+
+  const connectionInfo = { endpointUrl, clientVersion, specName, specVersion, latestBlockNumber };
+  console.log("\n## Connection Information");
+  console.table(Object.entries(connectionInfo).map(([key, value]) => ({ key, value: value.toString() })));
+
+  console.log("\n## Schema Information");
+  const maxSchemaId = await (await api.query.schemas.currentSchemaIdentifierMaximum()).toNumber();
```

Comment:
Why is it necessary to have an await (await ...)?

Reply:
It isn't! Removing...